idd/Energy+.idd.in

```diff
@@ -73923,6 +73923,321 @@ HeatPump:PlantLoop:EIR:Heating,
+       \note Enter the name of an OutdoorAir:Node object.
+  A5 , \field Companion Heating Heat Pump Name
+       \note The name of the companion HeatPump:AirToWater:FuelFired:Heating object
+       \note This field is used for a heat pump with switchable heating and cooling mode.
+       \type object-list
+       \object-list HeatPumpAirToWaterFuelFiredHeatingNames
+  A6 , \field Fuel Type
+       \required-field
+       \type choice
+       \key NaturalGas
+       \key Propane
+       \key FuelOilNo1
+       \key FuelOilNo2
+       \key Diesel
+       \key Gasoline
+       \key Coal
+       \key OtherFuel1
+       \key OtherFuel2
+       \default NaturalGas
+       \note Fuel Type (NaturalGas, Propane, Gasoline, Diesel etc.)
+  A7 , \field End-Use Subcategory
+       \type alpha
+       \retaincase
+       \default General
+       \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
+  N1 , \field Nominal Cooling Capacity
+       \autosizable
+       \minimum> 0
+       \units W
+       \note Nominal Cooling Capacity in [W] (autosizeable)
+  N2 , \field Design Flow Rate
+       \autosizable
+       \minimum> 0
+       \units m3/s
+       \note Design Flow Rate in m3/s (autosizeable)
+  N3 , \field Design Supply Temperature
+       \default 7.0
+       \units C
+       \note Design Supply Temperature in [degree C]
+  N4 , \field Design Temperature Lift
+       \autosizable
+       \default 11.1
+       \units deltaC
+       \note Design Temperature Lift in [degree C]
+  N5 , \field Sizing Factor
+       \minimum 1.0
+       \default 1.0
+       \note Sizing Factor for equipment sizing
+  A8 , \field Flow Mode
+       \required-field
+       \type choice
+       \key NotModulated
+       \key ConstantFlow
+       \key LeavingSetpointModulated
+       \default NotModulated
+       \note Flow Mode for the water side of the fuel-fired absorption heat pump
+  A9 , \field Outdoor Air Temperature Curve Input Variable
+       \required-field
+       \type choice
+       \key DryBulb
+       \key WetBulb
+       \default DryBulb
+       \note Outdoor air temperature curve input variable;
+       \note The options are Outdoor Air Dry Bulb or Wet Bulb temperature for curves
+  A10, \field Water Temperature Curve Input Variable
+       \required-field
+       \type choice
+       \key EnteringEvaporator
+       \key LeavingEvaporator
+       \default EnteringEvaporator
+       \note Water Temperature curve input variable - Condenser Entering or Leaving Water Temperature for curves
+  A11, \field Normalized Capacity Function of Temperature Curve Name
+       \required-field
+       \type object-list
+       \object-list BivariateFunctions
+       \note: CAPFT - Normalized Capacity Function of Temperature Curve name,
+       \note which is a biquadratic curve or a lookup table.
+  A12, \field Fuel Energy Input Ratio Function of Temperature Curve name
```

Comment:
Name

Reply:
Addressed in previous changes. Further, a minor change was just added for one note item to use "Name" as well in one occurrence in the note.